study.md

```diff
@@ -40,14 +40,19 @@ What does an exclamation point after a method usually signify?  e.g.,
 `my_string.capitalize!`
 
 ```md
-<!-- your answer here -->
+Dont make a copy and use the current instance.
 ```
 
 ## Instantiation
 How do you create an instance of a class in Ruby?
 
 ```ruby
-# your answer here
+class newClass
```

Comment:
This defines a new class but doesn't instantiate it.  `class_instance = ClassName.new`.